packages/merlin.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,34 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'Merlin'
+$category = 'Command & Control'
+$toolDir = Join-Path ${Env:RAW_TOOLS_DIR} $toolName
+
+$zipUrl = 'https://github.com/Ne0nd0g/merlin/releases/download/v1.5.1/merlinServer-Windows-x64.7z'
+$zipSha256 = 'e3c6ee205a46c9619dbe12bb4d487b7bcc802113658f29397b5f550295fc76fc'
+$zipPassword = 'merlin'
```

Comment:
It is the first time I see a tool zip protected, should we add the password to the `VM-Install-From-Zip` helper to avoid duplication?

Reply:
Same here. First time I've seen a tool zipped up like this. Seems like more of an edge case to me, especially because this is a 7z archive, which forced me to add it as a dependency.